### PR TITLE
Modify looping test Jenkins to patch out Rancher finalizers from install ns

### PR DESCRIPTION
**Pull request by michael-cico**
# Description

Modify looping test Jenkins to patch out Rancher finalizers from install ns
- also fix go fmt issue in tests

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by agv426 — APPROVED** [review]

File: ci/looping-test/Jenkinsfile [hunk 1]
```diff
@@ -338,7 +338,11 @@ pipeline {
             steps {
                 catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
                     sh """
-                        kubectl delete -f ${WORKSPACE}/acceptance-test-operator.yaml
+                        kubectl delete --wait=false -f ${WORKSPACE}/acceptance-test-operator.yaml
+                        echo "Workaround, removing any dangling finalizers from the install namespace"
+                        kubectl patch namespace verrazzano-install -p '{"metadata":{"finalizers":null}}' --type=merge || true
+                        kubectl wait --for=delete deployment/verrazzano-platform-operator --timeout=5m || true
```
Comment (bobdonat) [inline ci/looping-test/Jenkinsfile:344]: doesn't this `kubectl wait` command need to specify a namespace

Comment (michael-cico) [reply ci/looping-test/Jenkinsfile:344]: actually, I probably just need the wait below for the namespace to go away.  Is that what you mean?

Comment (bobdonat) [reply ci/looping-test/Jenkinsfile:344]: No, the wait for deployment/verrazzano-platform-operator needs to include "-n verrazzano-install"

Comment (michael-cico) [reply ci/looping-test/Jenkinsfile:344]: ah, right...but, I think I just need to wait for the namespace to delete when I look at it.  The yaml file has the ns in it, so we should just be able to wait on that deletion.

**Review by agv426 — APPROVED** [review]

**Pull request merged**